Sources/App/App+build.swift

```diff
@@ -10,47 +11,32 @@ import Logging
 import OpenAPIHummingbird
 {{/hbOpenAPI}}
 
-{{^hbLambda}}
-/// Application arguments protocol. We use a protocol so we can call
-/// `buildApplication` inside Tests as well as in the App executable. 
-/// Any variables added here also have to be added to `App` in App.swift and 
-/// `TestArguments` in AppTest.swift
-package protocol AppArguments {
-    var hostname: String { get }
-    var port: Int { get }
-    var logLevel: Logger.Level? { get }
-}
-{{/hbLambda}}
-
 // Request context used by {{^hbLambda}}application{{/hbLambda}}{{#hbLambda}}lambda<{{hbLambdaType}}Request>{{/hbLambda}}
 typealias AppRequestContext = {{^hbLambda}}BasicRequestContext{{/hbLambda}}{{#hbLambda}}BasicLambdaRequestContext<{{hbLambdaType}}Request>{{/hbLambda}}
 
 {{^hbLambda}}
 ///  Build application
-/// - Parameter arguments: application arguments
-func buildApplication(_ arguments: some AppArguments) async throws -> some ApplicationProtocol {
+/// - Parameter configReader: configuration reader
+func buildApplication(configReader: ConfigReader) async throws -> some ApplicationProtocol {
```

Comment:
```suggestion
func buildApplication(reader: ConfigReader) async throws -> some ApplicationProtocol {
```
To make it consistent with HB APIs